include/substrait/common/Io.h

```diff
@@ -0,0 +1,30 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+
+#pragma once
+
+#include <string_view>
+
+#include "absl/status/statusor.h"
+#include "substrait/proto/plan.pb.h"
+
+namespace io::substrait {
+
+enum PlanFileEncoding {
```

Comment:
Encoding makes me think of UTF-8, especially if we're talking about saving & loading.  Maybe we could use something with "format" in the name?

Reply:
I ended up with PlanFileFormat.